staff/carlos-calvo/cv/index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV Carlos Calvo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>CV CARLOS CALVO LOPEZ</header>
+    <img src=""></img>
+    <hr/>
+    <section>
+        <h1><b>PERSONAL INFORMATION</b></h1>
+        <ul>
+            <li>Name: Carlos</li>
+            <li>Surname: Calvo</li>
+            <li>Birth Date: 09/12/1986</li>
+            <li>mail: carloscl1986@gmail.com</li>
+            <li>Mobile Phone: 620092996</li>
+        </ul>
+    </section>
+    <hr/>
+    <section>
+        <h1><b>FORMATION</b></h1>
+        <ul>
+            <li>Industrial Engineering
+                <ul>
+                    <li>UPC, Barcelona</li>
+                    <li>2004 - 2010</li>
```

Comment:
Also, you can use the `<datetime>` tag to wrap the date & times:
`<time datetime="2004-01-01">2004</time> - <time datetime="2010-12-31">2010</time>`

Reply:
OK, that's better